shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -3,7 +3,28 @@
 // found in the LICENSE file.
 
 #include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterAppDelegate.h"
+#include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h"
 
 @implementation FlutterAppDelegate
 
+// Returns the key window's rootViewController, if it's a FlutterViewController.
```

Comment:
Let's add the documentation we talked about in case the user doesn't want to subclass.